doc/cug.tex

```diff
@@ -1236,35 +1236,31 @@ \subsection{Locating Suite And Task Output}
 \lstset{language=transcript}
 \begin{lstlisting}
 shell$ tree $HOME/cylc-run/tut.oneoff.basic/
-|-- cylc-suite.db      # suite run database
-|-- cylc-suite-env     # suite environment file
-|-- share              # suite share directory (not used in this example) 
-|-- work               # task work space (sub-dirs are deleted if not used)
-|    `-- 1                # task cycle point directory (or 1)
-|        `-- hello           # task work directory (deleted if not used)
-|-- log                # suite log directory
-|   `-- suite             # suite daemon log directory
-|   |   |-- err              # suite daemon stderr log (daemon mode only)
-|   |   |-- out              # suite daemon stdout log (damon mode only)
-|   |   `-- log              # suite daemon event log (timestamped info)     
-|   `-- job               # task job log directory
-|       `-- 1                # task cycle point directory (or 1)
-|           `-- hello           # task name
-|               |-- 01             # task submission number
-|               |   |-- job           # task job script
+|-- cylc-suite-private.db # private suite run database
```

Comment:
I quite like the idea of archiving dbs with the logs, but that said, cylc-suite.db is technically not just a log (of events etc.), so I'm not convinced that it logically belongs in the log directory.